modules/sandboxed-containers-deleting-kataconfig-cli.adoc

```diff
@@ -23,15 +34,13 @@ Remove and uninstall the `kata` runtime and all its related resources, such as C
 $ oc delete kataconfig <KataConfig_CR_Name>
 ----
 
-. Delete the `KataConfig` custom resource definition by running the following command:
-+
-[source,terminal]
-----
-$ oc delete crd kataconfigs.kataconfiguration.openshift.io
-----
-
 The {sandboxed-containers-operator} removes all resources that were initially created to enable the runtime on your cluster.
 
+[IMPORTANT]
+====
+During deletion, the Command Line Interface freezes until all worker nodes reboot. Wait for the process to complete before performing the verification or continuing to the next procedure.
```

Comment:
suggestion: CLI - spell out at first use, then use `CLI`.

IBM style guide: freeze: Do not use to refer to a software application that stops responding. For example, do not write "the application freezes". Instead, write "the application hangs" or "the application stops responding".

Reply:
Done.